components/dashboard/src/components/UsageView.tsx

```diff
@@ -160,13 +179,61 @@ function UsageView({ attributionId }: UsageViewProps) {
 
     const headerTitle = attributionId.kind === "team" ? "Team Usage" : "Personal Usage";
 
+    const FromLink = forwardRef((arg: any, ref: any) => (
+        <div
+            className="text-blue-500 hover:text-blue-600 dark:text-blue-400 dark:hover:text-blue-500 truncate cursor-pointer"
+            onClick={arg.onClick}
+            ref={ref}
+        >
+            {arg.value}
+        </div>
+    ));
+    const ToLink = forwardRef((arg: any, ref: any) => (
+        <div
+            className="text-blue-500 hover:text-blue-600 dark:text-blue-400 dark:hover:text-blue-500 truncate cursor-pointer"
+            onClick={arg.onClick}
+            ref={ref}
+        >
+            {arg.value}
+        </div>
+    ));
+
     return (
         <>
             <Header
-                title={headerTitle}
-                subtitle={`${new Date(startDateOfBillMonth).toLocaleDateString()} - ${new Date(
-                    endDateOfBillMonth,
-                ).toLocaleDateString()} (updated every 15 minutes).`}
+                title={
+                    <div className="flex items-baseline">
+                        <h1 className="tracking-wide">{headerTitle}</h1>
+                        <h2 className="ml-3">(updated every 15 minutes).</h2>
+                    </div>
+                }
+                subtitle={
+                    <h2 className="tracking-wide flex mt-3">
+                        <h2 className="mr-2">Period: </h2>
```

Comment:
**thought**: The idea of having the period selection on the top of the left sidebar is to update all usage entries on the main content as well as all aggregated data on the left sidebar based on the period selection. We could use the heading subtitle for hijacking this action and filtering other views like workpaces per team, etc.

| Idea of the top-left control for period selection | 
|-|
| <img width="1440" alt="Frame 1135" src="https://user-images.githubusercontent.com/120486/200920628-72b8545c-0c4f-462f-9547-f0f339b170e1.png"> |